samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb

```diff
@@ -58,18 +66,24 @@
    "metadata": {
     "dotnet_interactive": {
      "language": "fsharp"
+    },
+    "vscode": {
+     "languageId": "dotnet-interactive.fsharp"
     }
    },
    "outputs": [
     {
      "data": {
-      "text/plain": "[1; 2; 3; 4]\r\n"
+      "text/plain": [
+       "[1; 2; 3; 4]\r\n"
+      ]
      },
-     "output_type": "unknown"
+     "metadata": {},
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "Formatter.SetPreferredMimeTypeFor(typeof<obj>, \"text/plain\")\n",
+    "Formatter.SetPreferredMimeTypesFor(typeof<obj>, \"text/plain\")\n",
```

Comment:
`Formatter.SetPreferredMimeTypeFor` doesn't seem to exist anymore (note singular on `Type`, so I updated this.

I contemplated doing `Formatter.SetPreferredMimeTypesFor(typeof<obj>, [|"text/plain"|])`.